site/content/en/docs/drivers/qemu.md

```diff
@@ -21,6 +21,14 @@ To start minikube with the qemu driver:
 minikube start --driver=qemu
 ```
 
+## Special features
+
+minikube start supports some qemu specific flags:
+
+* **`--qemu-firmware-path`**: The path to the firmware image to be used.
+  * Note: while the flag should override the config, if the flag does not take effect try running `minikube delete --all --purge`.
```

Comment:
Could you explain why we need to delete the entire `~/.minikube` directory, this seems overkill. This flag takes effect at a profile level, so worst case only the individual profile should be removed.

`while the flag should override the config, if the flag does not take effect` Is there a reason the flag won't override the config? Is there something we can change in the code to make it consistently override so we don't have to have this note?

Reply:
In his [comment](https://github.com/kubernetes/minikube/issues/14622#issuecomment-1192602670), @vbezhenar found that only by removing `~/.minikube` was he able to make progress.

Instead of removing `~/.minikube`, @medyagh suggested `minikube delete --all --purge`.

The note is a suggestion to help others save time when debugging, but I agree that the code should work without the override. I have not been able to reproduce this.

In a later PR, I plan to specifically check for the qemu firmware path, if not available, I will also check if Qemu was installed via MacPorts instead.

Reply:
I just read the comment and I agree that deleting `~/.minikube` will resolve this, but that seems like more of a nuclear option. In the issue thread he never talks about `minikube delete`, likely because he wasn't aware of it, so he manually deleted all of `~/.minikube` instead. I'm sure a simple `minikube delete` would have sufficed since that would have deleted the entire config for the profile including `qemu-firmware-path` which is all he truly wanted removed but went about it a different way.

Reply:
Sure, that makes sense. In this case, is your preference:

1. Update the note to say `minikube delete`.
2. Remove the note altogether, as most users may already know `minikube delete`.

Reply:
1. makes more sense to me, and hopefully we can remove the note altogether in the future if we can guarantee that  the flag will always take effect.

Reply:
Yes, have updated the PR per 1. and I agree with hopefully removing the note altogether in the future.

Reply:
> I just read the comment and I agree that deleting `~/.minikube` will resolve this, but that seems like more of a nuclear option. In the issue thread he never talks about `minikube delete`, likely because he wasn't aware of it, so he manually deleted all of `~/.minikube` instead. I'm sure a simple `minikube delete` would have sufficed since that would have deleted the entire config for the profile including `qemu-firmware-path` which is all he truly wanted removed but went about it a different way.

You're right, I just did the first thing that came in my head as I just started using minikube and there wasn't anything important for me.